scripts/plugins/anchor-markdown-headings.js

```diff
@@ -1,14 +1,42 @@
 'use strict'
 
+// Check whether we've got the comment
+const ANCHOR_COMMENTREG = /<!--[\w-]+-->/gi
```

Comment:
`const ANCHOR_COMMENTREG = /<!--\x20?([\w-]+)\x20?-->/gi`

Matches `<!--comment-->`, `<!-- comment -->` and even `<!--  comment  -->`
(20 hex = 32 dec = space character)